### PR TITLE
Update labels and placeholder for server url

### PR DESCRIPTION
**Pull request by pirkla**
Update storyboard to use Jamf Pro Server URL instead of Jamf Pro Server for the label.
Update bindings for Jamf Pro Server URL placeholder values to use a sample url instead of [Required]

### CODE REVIEW

**Comment by hisaac** [comment]
Excuse me while I get a little UX-y here, but I have a thought on the text labels and placeholders:

The purpose of a placeholder is to give a hint toward the expected format of what the user is about to enter, or to act as a replacement of the label. [As mentioned here](https://github.com/jamf/PPPC-Utility/issues/32#issuecomment-568557590), the "[Required]" / "[Optional]" placeholders are non-standard.

Coming to this new, when I look at this screen, I'd assume that the only field that is *not* required is the "Payload Description" field. Is that the case?

Screen Shot 2020-02-10 at 4 47 48 PM

If that is the case, we could just remove all the [Required] and [Optional] placeholders, and add "(Optional)" to the end of the "Payload Description" label. That would allow the user to assume the rest of the fields are non-optional.

And while we're at it, rather than having placeholders in the Username and Password fields, we could change their labels to be "Jamf Pro Username" and "Jamf Pro Password" respectively.

What do you folks think?

**Comment by pirkla** [comment]
I agree, and it looks like the default placeholder text for all fields doesn't show required, it's just the binding is set to show [Required] as a null placeholder. That can cause some odd behavior since you'll see the other placeholder until the value is modified and the binding updates.
placeholder

I think we might actually get the intended behavior by fixing that.

The label changes wouldn't hurt to throw in the update while we're at it. I was worried it would look wordy but on a full screen it's not so bad. I'll update the branch with some good stuff for review.

File: Resources/Base.lproj/Main.storyboard [hunk 1]
```diff
@@ -2771,10 +2771,10 @@
                                                         <string key="NSValueTransformerName">NSIsNil</string>
                                                     </dictionary>
                                                 </binding>
-                                                <binding destination="myc-4h-VBq" name="value" keyPath="values.jamfProServer" id="XiN-F4-3RS">
+                                                <binding destination="myc-4h-VBq" name="value" keyPath="values.jamfProServer" id="b2i-mX-P1C">
                                                     <dictionary key="options">
                                                         <bool key="NSContinuouslyUpdatesValue" value="YES"/>
-                                                        <string key="NSNullPlaceholder">[Required]</string>
+                                                        <string key="NSNullPlaceholder">https://jss.mycompany.com:8443/</string>
```
[the remark this answers is not in the document]
Comment (hisaac) [reply Resources/Base.lproj/Main.storyboard]: I don't really have a preference either way, so I'm fine with whichever.

[the remark this answers is not in the document]
Comment (mm512) [reply Resources/Base.lproj/Main.storyboard]: I'm good with using https://example.jamfcloud.com as a placeholder.

**Pull request merged**